ui/src/Room.tsx

```diff
@@ -252,6 +293,19 @@ export const Room = ({
                         keepMounted
                         open={Boolean(showMore)}
                         onClose={() => setShowMore(undefined)}>
+                        <MenuItem onClick={() => setVideoDisplayMode(VideoDisplayMode.FitToWindow)}>
```

Comment:
Could you move the settings to a separate menu, which can be opened by clicking an entry like "Display Mode" inside the "triple dot" menu? I can change this too, if you don't have the time for this (:. Also, it would be great, if the menu would automatically close after an option is selected.